chore: address commit comments from bc90eba (#5861)

JaySoni1 opened the pull request:
Resolves #5861

## Description

> What is the purpose of this pull request?

This pull request:

-   resolves- Address commit comments (commit bc90eba) #5861

## Related Issues

> Does this pull request have any related issues?

This pull request:

-   resolves #5861

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

JaySoni1 committed:
chore: fixed C lint errors

JaySoni1 committed:
chore: fixed C lint errors

JaySoni1 committed:
chore: fixed C lint errors

JaySoni1 committed:
chore: fixed C lint errors

JaySoni1 committed:
chore: fixed C lint errors

JaySoni1 committed:
chore: fixed C lint errors

JaySoni1 committed:
chore: fixed C lint errors

JaySoni1 committed:
chore: fixed C lint errors

JaySoni1 committed:
Merge branch 'develop' into issue-5861

Signed-off-by: JaySoni1 <jaygsoni14@gmail.com>

JaySoni1 commented:
@kgryte , @Planeshifter please review

JaySoni1 commented:
@kgryte , @Planeshifter please review

kgryte commented:
Unfortunately, this PR contains many unwanted changes and cannot be merged as is. As such, I am going to go ahead and close.

JaySoni1 commented:
Thanks for the clarification @kgryte . I’ll create a fresh PR addressing the comments, ensuring only the intended changes are included. I’ll also make sure the commit history is clean. Will request a review once it’s ready!
